main.js

```diff
@@ -30,40 +22,24 @@ class InitLayer {
 // lancement des web worker pour remonter les datas
 
 const giveData = new Worker("info.js");
+let federations = {};
```

Comment:
I'm agree with you for the final solution and with the name "federations" (I'll renamed it federationsLayer) 
but the final object that I created to do is :
```
federationsLayer : {
    "BA": BA,
    "ON":ON,
    ...
}
```
I need that because the [exemple](http://leafletjs.com/examples/layers-control/) expect to me to return this object where `BA` or `ON` is a variable which contains `L.layerGroup().addLayer(L.marker([lat.city1, lng.city1]))..addLayer(L.marker([lat.city2, lng.city2])).addLayer(L.marker([lat.city3, lng.city3]))` and so on